### DISCUSSION TITLE
SQLite store

### DISCUSSION THREAD

**Discussion by nfriedly** [nfriedly/express-rate-limit#475] [Ideas]
Node.js recently added a [built-in SQLite module](https://nodejs.org/api/sqlite.html), and I'm thinking about creating a SQLite store that takes advantage of it. It would make sense for similar use-cases to the memory store and cluster memory store, but with the benefits of remembering state across restarts and better compatibility with PM2 and other process managers.

(Also, I just like SQLite ;)

**Comment by gamemaker1** [comment]
That's a great idea, we should definitely do this.
